e2e/regression/guardrail.insulin_sensitivities.spec.js

```diff
@@ -0,0 +1,43 @@
+const { LoopTest, screenName, target } = require('../../src/index');
+
+describe('guardrail insulin sensitivities', () => {
+    var loopTest;
+    it('should setup with pump simulator', async () => {
+        loopTest = await new LoopTest.Builder(target.tidepool)
+            .withSettings({ AddPumpSimulator: true })
+            .withStartScreen(screenName.settings)
+            .build();
+    });
+
+    it('set 500 units, warning', async () => {
+        await loopTest.settingsScreen.SetInsulinSensitivities([{ time: '12:00 AM', bgValuePerInsulinUnit: '500' }]);
+        //TODO assert on warning
+    });
+    it('set 400 units, warning', async () => {
+        await loopTest.settingsScreen.SetInsulinSensitivities([{ time: '12:30 AM', bgValuePerInsulinUnit: '400' }]);
+        //TODO assert on warning
+    });
```

Comment:
What about tests for 399 and 16 (or whatever are the values that just barely don't give you a warning on the high and low sides)?

Reply:
Basically, I think when testing boundary conditions you want to test both sides of the boundary.